libs/designer-ui/src/lib/pager/index.tsx

```diff
@@ -186,44 +216,109 @@ export const Pager: React.FC<PagerProps> = ({
     description: 'Button indicating to go to the next page',
   });
 
+  const pageNumbers = useMemo(() => {
+    const result = [];
+
+    // Calculate initial range around the current number
+    let rangeStart = Math.max(current - 2, min);
+    let rangeEnd = Math.min(current + 2, max);
+
+    // Adjust the range if it's less than 5 numbers
+    while (rangeEnd - rangeStart + 1 < 5) {
```

Comment:
I am assuming 5 is the default page number you want to show.. can you keep this as a constant or take that as a prop, i am assuming based on the caller and teh area they have they can determine how much to show..
since this is UI element lets try to move this to props so it is configurable.

Reply:
Based on the portal Pager UI that we are using as a reference, it did not provide the flexibility to adjust the number of pages shown. I thought there would be a good reason for that, but since this is our custom component, I can move it as a prop with a default value of 5